include/misc/util.h

```diff
@@ -373,6 +373,222 @@ static inline s64_t arithmetic_shift_right(s64_t value, u8_t shift)
 #define UTIL_DEC_38 37
 #define UTIL_DEC_39 38
 #define UTIL_DEC_40 39
+#define UTIL_DEC_41 40
```

Comment:
Can this be reworked to not require a #define at all?

Reply:
Could you suggest something specific? How should I go about this?

Reply:
Could you refactor it in independent PR? It seems you have a clear idea of how to approach this.